src/global_mapper_ros/global_mapper_ros.h

```diff
@@ -3,9 +3,11 @@
 
 #include <csignal>
 #include <memory>
+#include <string>
+#include <vector>
 
 #include "ros/ros.h"
-#include <tf2_ros/transform_listener.h>
+#include "tf2_ros/transform_listener.h"
```

Comment:
Alright, this one is satisfying.  Moving on with life.
https://gcc.gnu.org/onlinedocs/cpp/Include-Syntax.html